sklearn/cluster/_agglomerative.py

```diff
@@ -384,16 +384,16 @@ def linkage_tree(X, connectivity=None, n_clusters=None, linkage='complete',
             - single uses the minimum of the distances between all observations
               of the two sets.
 
-    affinity : string or callable, optional, default: "euclidean".
+    affinity : string or callable, default="euclidean".
```

Comment:
It is preferred to use Python basic types. In this case, `string` should be `str`.

Reply:
ok